Geometry_Engine/Query/DiscontinuityPoints.cs

```diff
@@ -55,15 +55,6 @@ public static List<Point> DiscontinuityPoints(this Line curve)
 
         /***************************************************/
 
-        [NotImplemented]
-        public static List<Point> DiscontinuityPoints(this NurbsCurve curve)
-        {
-            throw new NotImplementedException();
-        }
-
-        /***************************************************/
-
-        [NotImplemented]
```

Comment:
Not sure I would remove this tbh. DO not think this method is up to scratch. Tangents needs to be checked between the start/end of a segment:

![image](https://user-images.githubusercontent.com/22005920/91832465-b9a94c00-ec45-11ea-8eea-114628aa7369.png)

top is as PolyCurve, bottom is as Polyline.

Would rather return an exception here.

Reply:
Great point!
I will keep the attribute for now for the sake of PR's clarity and fix that one together with correcting inputs for this and other `DiscontinuityPoints` methods right after this PR got merged.